### PR TITLE
systemd: ceph-mgr: set MemoryDenyWriteExecute to false

### PR DESCRIPTION
**Pull request by rjfd**
Fixes: http://tracker.ceph.com/issues/39628

Signed-off-by: Ricardo Dias <rdias@suse.com>

- [x] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

### CODE REVIEW

**Comment by LenzGr** [comment]
For the record, this change was introduced in #26248

Adding the dashboard label, as it is affected by this issue.

**Comment by LenzGr** [comment]
Do we have any QA test suite that actually uses systemd to spawn the services?

File: systemd/ceph-mgr@.service.in [hunk 1]
```diff
@@ -12,7 +12,7 @@ Environment=CLUSTER=ceph
 ExecStart=/usr/bin/ceph-mgr -f --cluster ${CLUSTER} --id %i --setuser ceph --setgroup ceph
 ExecReload=/bin/kill -HUP $MAINPID
 LockPersonality=true
-MemoryDenyWriteExecute=true
+MemoryDenyWriteExecute=false
```
Comment (batrick) [inline systemd/ceph-mgr@.service.in]: Please add a comment that this is needed for FFI/JIT.

**Comment by callithea** [comment]
FYI:
http://pulpito.ceph.com/laura-2019-05-08_21:22:00-rados:mgr-wip-lpaduano-testing-28023-distro-basic-mira/

File: systemd/ceph-mgr@.service.in (hunk 1 above)
Comment (rjfd) [reply systemd/ceph-mgr@.service.in]: done

**Comment by smithfarm** [comment]
> Do we have any QA test suite that actually uses systemd to spawn the services?

Not sure, but possibly `--suite smoke` is what you're looking for?

**Comment by callithea** [comment]
Triggered another test run (with `--suite smoke/systemd`):
http://pulpito.ceph.com/laura-2019-05-15_09:31:34-smoke:systemd-wip-lpaduano-testing-28023-2-distro-basic-mira/

**Comment by callithea** [comment]
jenkins test make check

**Comment by callithea** [comment]
jenkins test make check

**Comment by smithfarm** [comment]
@callithea This is good to go. You can ignore the "make check (arm64)" failure.

**Comment by callithea** [comment]
> @callithea This is good to go. You can ignore the "make check (arm64)" failure.
> 
> Also, the failures in `smoke/systemd` should be compared with previous runs of the same suite... Is it possible the suite itself is broken? In other words, is it known to be passing on tip of master?

@smithfarm okay, thanks for letting me know.
I would need to test the suite again based on current master.
When filtering for that suite in pulpito it seems like it hasn't been used in a long time and a lot of the test runs failed.

**Comment by smithfarm** [comment]
> When filtering for that suite in pulpito it seems like it hasn't been used in a long time and a lot of the test runs failed.

Then it probably isn't useful for detecting regressions in PRs.

**Pull request merged**